src/components/dashboard/Dashboard.jsx

```diff
@@ -1,18 +1,25 @@
 import React from 'react';
 import NavBar from '../navbar/NavBar';
-import "../dashboard/dashboard.css"
-import Dashboardbg from "../../assets/dashboard_background.jpg"
 
+import Slideshow from './SlideShow';
+import chef_wallpaper1 from '../../assets/chef_wallpaper1.jpg';
+import packed_food from '../../assets/packed_food.jpg';
+import chef_wallpaper2 from '../../assets/chef_wallpaper2.jpg';
+import Delivery_wallpaper from '../../assets/Delivery_wallpaper.jpg';
```

Comment:
You're not consistent with your naming. This image name starts with upper case when previous ones start with lower case.